core/src/main/scala/cats/instances/sortedSet.scala

```diff
@@ -92,12 +92,11 @@ private[instances] trait SortedSetInstancesBinCompat0 {
 }
 
 private[instances] trait SortedSetInstancesBinCompat1 extends LowPrioritySortedSetInstancesBinCompat1 {
-  // TODO: Remove when this is no longer necessary for binary compatibility.
-  // Note that the overrides here and below are only necessary because the
-  // definitions in `SortedSetInstances1` conflict with the ones in
-  // `cats.kernel.instances.SortedSetInstances`. Both are inherited here, so
-  // we have to "bubble" the "correct" ones up to the appropriate place.
-  implicit override def catsKernelStdHashForSortedSet[A: Hash]: Hash[SortedSet[A]] =
+  implicit def catsKernelStdHashForSortedSet1[A: Hash]: Hash[SortedSet[A]] =
+    cats.kernel.instances.sortedSet.catsKernelStdHashForSortedSet[A](Hash[A])
+
+  @deprecated("Use cats.kernel.instances.sortedSet.catsKernelStdHashForSortedSet", "2.0.0-RC3")
+  override def catsKernelStdHashForSortedSet[A: Order: Hash]: Hash[SortedSet[A]] =
     cats.kernel.instances.sortedSet.catsKernelStdHashForSortedSet[A](Hash[A])
```

Comment:
It should be easy to add a test to binCompatTest for the original breakage right?

Reply:
I don't think so. I don't know of a way to make it show up without referring to `AllInstancesBinCompat6`, which was introduced after 1.6.1, which `binCompatTest` uses.